src/main/java/com/kramekk1/medicalClinic/PatientService.java

```diff
@@ -40,4 +40,10 @@ public void deletePatient(String email) {
         patientRepository.deletePatient(email);
     }
 
+    public void editPasswordByEmail(String email, String newPassword) {
+        Optional<Patient> patient = patientRepository.findPatientByEmail(email);
```

Comment:
.orElseThrow( () -> new IllegalArgumentException("Patient with given email does not exist"))